File: common/src/main/java/com/iflytek/common/ContextProvider.java
package com.iflytek.common;

import android.annotation.SuppressLint;
import android.content.Context;

/**
 * @author aljiang
 */
public class ContextProvider {
    @SuppressLint("StaticFieldLeak")
    private static Context context;

    public static void init(Context context) {
        ContextProvider.context = context;
    }

    public static Context getContext() {
        return context;
    }
}

File: app/src/main/java/com/jal/todo/app/ContextProvider.kt
package com.jal.todo.app

import android.content.Context

/**
 * @author aljiang
 * @date 2021/1/28
 * @desc
 */
object ContextProvider {
    lateinit var mContext: Context

    fun init(context: Context) {
        this.mContext = context
    }
}